assets/js/components/PresenterStreams.tsx

```diff
@@ -15,7 +15,7 @@ type PresenterStreamAreaProps = {
 };
 
 const playerCallbacks: { [key: string]: (sourceType: SourceType) => void } = {};
-let webrtc: MembraneWebRTC | null = null;
```

Comment:
This solution with boolean was mostly for debugging reasons. It is a hack and should be changed.